### ISSUE TITLE
Link editor

### ISSUE THREAD

**Issue by k88hudson** [mozilla/webmaker-app#1645]
The link editor still needs to be implemented on the page editor (which is currently `projects/projects.jsx`)

![image](https://cloud.githubusercontent.com/assets/1455535/7480048/b35e6102-f333-11e4-9074-342e4e879bc2.png)
(https://projects.invisionapp.com/share/BK2M18Q28#/screens/75741135)

Note that the actual link destination editor should be omitted, replaced with an alert (this is not done yet etc.)

**Comment by xmatthewx** [comment]
Depends on #1549

**Comment by k88hudson** [comment]
@xmatthewx is that issue not finalized?

**Comment by xmatthewx** [comment]
@vazquez revised today, but I haven't had a chance to review the changes. This is listed as Beta 1/1 on roadmap. It's great if we can land this for Alpha, but I'm guessing the destination picker will require a bit of time.

**Comment by xmatthewx** [comment]
...a bit of time to implement.

**Comment by k88hudson** [comment]
Ah, ok. What is the plan for what to do for v1 then? Omit/hide the link element from the add menu etc.?
cc @thisandagain

**Comment by thisandagain** [comment]
Let's show with alert (same as image selector). Hiding will make it seem "complete" which will likely raise a lot of questions and sets a lower expectation for the finished product.

**Comment by xmatthewx** [comment]
This will include:
 - element editor for button style and label
 - destination picker

Build of tinker mode for destination will come later.

For alpha, it'd be nice to have the editor if it's not a major lift.

**Comment by k88hudson** [comment]
Got it. I'll update the first comment.

**Comment by k88hudson** [comment]
done
